### ISSUE TITLE
Docs not up-to-date for the latest version of yq

### ISSUE THREAD

**Issue by lagalbra** [linkerd/linkerd2#5801]
# Thanks for your help improving the project! #

## Getting Help ##

Github issues are for bug reports and feature requests. For questions about
Linkerd, how to use it, or debugging assistance, start by
[asking a question in the forums](https://discourse.linkerd.io/) or join us on
[Slack](https://slack.linkerd.io/).

Full details at [CONTRIBUTING.md](https://github.com/linkerd/linkerd/blob/main/CONTRIBUTING.md).

## Filing a Linkerd issue ##

Issue Type:

- [X] Bug report
- [ ] Feature request

**What happened**:
linkerd commands related to retrieving the trust anchor certificate are not up-to-date with yq version 0.4.5 (as documented in linkerd.io/content/2/tasks/installing-multicluster.md and linkerd.io/content/2/tasks/manually-rotating-control-plane-tls-credentials.md):

```
$ kubectl -n linkerd get cm linkerd-config -o=jsonpath='{.data.values}'   | yq r - global.identityTrustAnchorsPEM > original-trust.crt
Error: unknown command "r" for "yq"
Run 'yq --help' for usage.
```

Note: added `.` in path is neccessary:

```
$ kubectl -n linkerd get cm linkerd-config -ojsonpath="{.data.values}" | yq e global.identityTrustAnchorsPEM - > trustAnchor.crt
Error: Parsing expression: Lexer error: could not match text starting at 1:1 failing at 1:2.
        unmatched text: "g"
$ kubectl -n linkerd get cm linkerd-config -ojsonpath="{.data.values}" | yq e .global.identityTrustAnchorsPEM - > trustAnchor.crt
$
```

**What you expected to happen**:
No error, certificate is written out to the file

**How to reproduce it (as minimally and precisely as possible)**:
1. Install latest yq version (`snap install yq` or following steps [here](https://github.com/mikefarah/yq#install))
2. Run step documented at https://linkerd.io/2/tasks/manually-rotating-control-plane-tls-credentials/#rotating-the-trust-anchor: `kubectl -n linkerd get cm linkerd-config -o=jsonpath='{.data.values}' | yq r - global.identityTrustAnchorsPEM > original-trust.crt`

**Anything else we need to know?**:
I have a fix ready at https://github.com/lagalbra/website/tree/update-rotate-doc-with-yq-version

**Environment**:
- linkerd/namerd version, config files:
    ```
    $ linkerd version
    Client version: stable-2.8.1
    Server version: stable-2.9.1
    ``` 
- Platform, version, and config files (Kubernetes, DC/OS, etc): N/A
- Cloud provider or hardware configuration: N/A

**Comment by adleong** [comment]
Thanks @lagalbra!  Are you interested in submitting a fix for this?

**Comment by lagalbra** [comment]
@adleong I would be happy to! Can I simply open a PR with the changes and reference this validated issue?

**Comment by kleimkuhler** [comment]
Looks like this will be fixed by linkerd/website#989

**Comment by lagalbra** [comment]
Fixed; see https://github.com/linkerd/website/pull/989
